### PR TITLE
Add Control Page for sending commands

### PR DESCRIPTION
**Pull request by AlyNat75**

### CODE REVIEW

**Comment by AlyNat75** [comment]
* **#115** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/115?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @AlyNat75 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: ground-server/src/components/actuation-box.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    setIsOn(false);
+  };
+
+  return (
+    <div className="border border-gray-300 p-6 rounded-lg bg-white dark:bg-black text-center">
+      <h2 className="text-lg font-bold mb-2">{title}</h2>
+      <div>
+        <h4 className="text-sm font-medium mb-2">Current Telemetry</h4>
+        <div 
+          className={`mb-4 h-[100px] ${theme === "dark" ? "bg-gray-800" : "bg-gray-200"}`}>
```
Comment (maxslarsson) [inline ground-server/src/components/actuation-box.tsx:41]: Try not to do this, instead replace `${theme === "dark" ? "bg-gray-800" : "bg-gray-200"}` with `bg-gray-200 dark:bg-gray-800`. I think this was the cause of the flickering and wrong color during code review

**Review by maxslarsson — CHANGES_REQUESTED** [review]

**Review by maxslarsson — APPROVED** [review]
Looks great!

**Comment by AlyNat75** [comment]
### Merge activity

* **Oct 17, 1:34 PM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/115).

**Pull request merged**